### DISCUSSION TITLE
Adding Drag and Drop features to the pieces

### DISCUSSION THREAD

**Discussion by ashishpapanai** [ashishpapanai/chessJS#98] [Ideas]
**JQuery offers certain tools to get the source i.e. the position from where the user started to drag to the position where the user has left the object.** 
_ChessJS expects the participant to get the start and the end co-ordinates of the drag and drop so that the maintainers can work on implementing this feature._ 

> People prefer dragging and dropping the pieces from the source to the destination of the piece to clicking the piece and the destination square.

**Comment by iamkrati22** [comment]
Would like to work on this issue

**Comment by ashishpapanai** [comment]
Great @iamkrati22 Please work on the master branch.

**Comment by ashishpapanai** [comment]
Please share your approach on slack @iamkrati22
